### PR TITLE
generator script: Update finding enums

### PR DESCRIPTION
**Pull request by ziga-lunarg**
This makes sure only enums from core and vulkan extensions are found

### CODE REVIEW

**Review by mark-lunarg — APPROVED** [review]

**Comment by aqnuep** [comment]
The proposed changes seem fine, but you should also change all other places where the "feature" and "extension" elements are referenced in the XPaths.

**Review by aqnuep — CHANGES_REQUESTED** [review]
Also, as these likely change e.g. the generated schema, it would be good to include any such generated content changes in the PR before merging.

File: scripts/genvp.py [hunk 1]
```diff
@@ -1391,7 +1391,7 @@ def parseVersionInfo(self, xml):
 
     def parseExtensionInfo(self, xml):
         self.extensions = dict()
-        for ext in xml.findall("./extensions/extension"):
+        for ext in xml.findall("./extensions/extension[@supported='vulkan']"):
             # Only care about enabled extensions
             if ext.get('supported') == "vulkan":
```
Comment (aqnuep) [inline scripts/genvp.py:1396]: Now this explicit if-check on the attribute value can be eliminated as the XPath filters out any other possibilities.

**Review by aqnuep — COMMENTED** [review]
Other than that minor refactoring comment, this looks fine.

File: scripts/genvp.py (hunk 1 above)
Comment (ziga-lunarg) [reply scripts/genvp.py]: Good catch, thanks!

**Pull request merged**